test/unit/core/viz/expressions/torque.test.js

```diff
@@ -1,32 +1,80 @@
 import * as s from '../../../../../src/core/viz/functions';
 import { validateTypeErrors, validateStaticType, validateFeatureDependentErrors } from './utils';
 
-describe('src/core/viz/expressions/torque', () => {
+fdescribe('src/core/viz/expressions/torque', () => {
     describe('error control', () => {
         validateFeatureDependentErrors('torque', [0.5, 'dependent']);
         validateTypeErrors('torque', ['category', 10]);
         validateTypeErrors('torque', ['number', 10, 'color']);
         validateTypeErrors('torque', ['color', 10]);
         validateTypeErrors('torque', ['number', 'color']);
     });
+
     describe('type', () => {
         validateStaticType('torque', ['number'], 'number');
         validateStaticType('torque', ['number', 10], 'number');
     });
+
     describe('.eval()', () => {
         it('should eval to 0 when the input is 1', () => {
             expect(s.torque(1).eval()).toEqual(0);
         });
+
         it('should eval close to 1 when the input is 0 and the fading is high', () => {
             const t = s.torque(0, 10, s.fade(10));
             t._setTimestamp(0);
             expect(t.eval()).toEqual(1);
         });
+
         it('should eval close to 0.75 when the input is 0 and we have wait a quarter of the animation', () => {
             const t = s.torque(0, 1, s.fade(1));
             t._setTimestamp(0);
             t._setTimestamp(0.25);
             expect(t.eval()).toEqual(0.75);
         });
     });
+
+
+
+    describe('.pause', () => {
+        it('should pause the simulation when playing', () => {
+            const t = s.torque(1, 10, s.fade(0));
```

Comment:
With a `0` fade is actually very difficult to don't pass the test. Please, check how the `should eval close to 0.75 when the input is 0 and we have wait a quarter of the animation` test does it

Reply:
we are only testing the how the torque progress changes when paused/stopped. I think the fade has no effect on it

Reply:
Having a zero fade is not good. Firstly, it is highly unrealistic since it basically means to not show anything, making torque() an alias to `0`. Secondly, you are checking that the `eval` returns 0 when using a fade that will eval to `0` in basically all circumstances, it is very easy to break the implementation and pass the tests.

Reply:
> you are checking that the eval returns 0

where?

Reply:
https://github.com/CartoDB/carto-vl/pull/550/files/bd83d3d483e0df08f73d27d6bd09b7ce4d20049a#diff-e3c5a624af6a9edc311887d558660ca5R45